dockstore-webservice/src/main/java/io/dockstore/webservice/DockstoreWebserviceConfiguration.java

```diff
@@ -102,6 +102,9 @@
     @NotEmpty
     private String discourseKey;
 
+    @NotNull
```

Comment:
Curious, doesn't have to block this PR
but if this is @NotNull, does it actually make sense to use Integer as opposed to int?

Reply:
I wasn't actually able to get int working.